aten/src/ATen/core/boxing/KernelFunction_impl.h

```diff
@@ -0,0 +1,177 @@
+#include <ATen/core/boxing/boxing.h>
+
+namespace c10 {
+
+inline KernelFunction::KernelFunction()
```

Comment:
Are there any changes to this file?

Reply:
`callUnboxed` now calls into `impl::boxAndCallBoxedFunc` instead of `detail::boxAndCallBoxedFunc` and that function implementation moved into its own header `boxing.h`. That's also the reason why this whole code movement happens - there's a dependency cycle between `KernelFunction.h` and `boxing.h` that I'm resolving by creating a `KernelFunction_impl.h`.

Reply:
OK... but did you change any of the code in this file? :)

Reply:
This file is new and it has all the `KernelFunction` methods. The code inside the methods is unchanged except for what I mentioned above.